### ISSUE TITLE
Ability to specify the package source

### ISSUE THREAD

**Issue by danielmarbach** [ParticularLabs/APIComparer#27]
Discussed with @adamralph it would be cool to be able to specify the package source. I think we could restrict ourself to only public sources like Nuget and MyGet.

How would the URI schema look like? @SeanFeldman and I agree that we need to go back to the drawing board to also solve #23

**Comment by adamralph** [comment]
This would be awesome. The use case is comparing the current release to the current pre-release in myget.

I like the way the comparisons are addressable, but I guess it could get quite messy if you had to have an encoded URL in the URL :stuck_out_tongue_closed_eyes:

I guess one way is to have defined aliases,

e.g. 'nuget' => nuget gallery feed URL, 'myget/myfeed' = myget root URL/myfeed

but it would be nicer, of course, to work against *any* package source.

**Comment by danielmarbach** [comment]
@adamralph as soon as we need package feed authentication it gets complicated pretty quickly. I think we need to start small

**Comment by SeanFeldman** [comment]
+1 for this info @adamralph 
Any package requirement can get tricky - private feeds would require authentication.
What we need (discussed with @danielmarbach) is to define what APIComparer will or will not provide.

**Comment by adamralph** [comment]
Yeah, agreed. No point trying to solve the world straight away. A specific scheme to just support myget would be good initial enhancement.

**Comment by andreasohlund** [comment]
Side note: this already works for the CLI which is very handy when you want to check your pull/feature branch for diffs against latest build:

https://github.com/ParticularLabs/APIComparer/blob/master/APIComparer/Program.cs#L18

**Comment by danielmarbach** [comment]
Yeah but the API is the new console and the API doesn’t support it

**Comment by andreasohlund** [comment]
I said it was a side note :)
